index.html

```diff
@@ -144,10 +144,48 @@
 
 		// Set style for ways
 		var EwaysStyle = {
-			color: "#E22B2B", 
+			color: "#46e370", 
+			weight: 5, 
+			opacity: 0.70
+		};
+		
+		var footwayStyle = {
+			color: "MediumOrchid", 
+			weight: 5, 
+			opacity: 0.70
+		};
+		
+		var residentialStyle = {
+			color: "MediumSlateBlue", 
 			weight: 5, 
 			opacity: 0.70
 		};
+		
+		var serviceStyle = {
+			color: "Wheat", 
+			weight: 5, 
+			opacity: 0.70
+		};
+		
+		var primaryStyle = {
+			color: "#Lavender", 
+			weight: 5, 
+			opacity: 0.70
+		};
+		
+		style: function(feature) {
```

Comment:
@SlpyBkWrm Great work on creating the styles! Move this block of code down to line 192 to set the style for the layer and be sure to change it to `feature.properties.highway` and the other ones in the line of code, since that is where the information is stored.